src/arrayMethodSort.js

```diff
@@ -3,9 +3,29 @@
 /**
  * Implement method Sort
  */
+/*
+realise sort method for array
+which repeats the original sort method
+the beginning is already there
+*/
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    String(a) > String(b)) {
```

Comment:
let's align your code
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) =>
    String(a) > String(b)
  ) {
```

Reply:
done